docs/fleet/fragments.md

```diff
@@ -9,6 +9,10 @@ aliases:
   - /fleet/configure-a-fleet/
 ---
 
+{{< cards >}}
```

Comment:
I think this needs to have some intro

Reply:
eh?
```suggestion
_This page contains reference information.
For a how-to guide, click here:_

{{< cards >}}
```

Reply:
Or, a la data capture:
```suggestion
Get started with a how-to guide or keep reading for more details.

{{< cards >}}
```

Reply:
I think the second but maybe after line 39?

Reply:
Okay. Adding a heading back in too for less confusion.